index.html

```diff
@@ -404,10 +428,13 @@ <h3><dfn>DataFactory</dfn> interface</h3>
       parameter is undefined a new identifier for the blank node is generated for each call.
     </p>
     <p>
-      <dfn>literal()</dfn> returns a new instance of <code>Literal</code>. If
-      <code>languageOrDatatype</code> is a <code>NamedNode</code>, then it is used for the value of
-      <code>datatype</code>. Otherwise <code>languageOrDatatype</code> is used for the value of
-      <code>language</code>.
+      <dfn>literal()</dfn> returns a new instance of <code>Literal</code>.
+      If <code>languageOrDatatype</code> is a <code>NamedNode</code>,
+      then it is used for the value of <code>datatype</code>.
+      If <code>languageOrDatatype</code> is a <code>DirectionalLanguage</code>,
+      then its <code>language</code> and <code>direction</code> attributes
+      are respectively used for the literal's <code>language</code> and <code>direction</code>.
```

Comment:
```suggestion
      are respectively used for the literal's <code>language</code> and <code>direction</code>, where <code>direction</code> is optional or can be falsy.
```

Just to make it clearer, that direction is optional or can be falsy.